apps/generator/package.json

```
@@ -0,0 +1,90 @@
+{
+  "name": "@asyncapi/generator",
+  "version": "2.0.1",
+  "description": "The AsyncAPI generator. It can generate documentation, code, anything!",
+  "main": "./lib/generator.js",
+  "bin": {
+    "asyncapi-generator": "./cli.js",
+    "ag": "./cli.js"
+  },
+  "engines": {
+    "node": ">=18.12.0",
+    "npm": ">=8.19.0"
+  },
+  "scripts": {
+    "test": "npm run test:unit && npm run test:integration && npm run test:cli",
+    "test:unit": "jest --coverage --testPathIgnorePatterns=integration --testPathIgnorePatterns=test-project",
+    "test:dev": "npm run test:unit -- --watchAll",
+    "test:integration": "npm run test:cleanup && jest --testPathPattern=integration --modulePathIgnorePatterns='./__mocks__'",
+    "test:integration:update": "jest --updateSnapshot --testPathPattern=integration --modulePathIgnorePatterns='./__mocks__'",
+    "test:cli": "node cli.js ./test/docs/dummy.yml ./test/test-templates/react-template -o test/output --force-write --debug && test -e test/output/test-file.md",
+    "test:cleanup": "rimraf \"test/temp\"",
+    "docs": "jsdoc2md --partial docs/jsdoc2md-handlebars/custom-sig-name.hbs docs/jsdoc2md-handlebars/main.hbs docs/jsdoc2md-handlebars/docs.hbs docs/jsdoc2md-handlebars/header.hbs docs/jsdoc2md-handlebars/defaultvalue.hbs docs/jsdoc2md-handlebars/link.hbs docs/jsdoc2md-handlebars/params-table.hbs --files lib/generator.js > docs/api.md",
+    "docker:build": "docker build -t asyncapi/generator:latest .",
+    "lint": "eslint --max-warnings 0 --config ../../.eslintrc .",
```

```suggestion
    "lint": "eslint --max-warnings 0 --config ../../.eslintrc --ignore-path ../../.eslintignore .",
```

linting still failing, maybe in CI it still needs proper explicit config for eslintignore